src/test/java/com/ldbc/driver/workloads/ldbc/snb/bi/BiWorkloadTest.java

```diff
@@ -397,6 +404,9 @@ public void shouldConvertFrequenciesToInterleavesWhenFrequenciesProvidedAndSomeI
             assertThat(
                     configurationAsMap.get( LdbcSnbBiWorkloadConfiguration.OPERATION_24_INTERLEAVE_KEY ),
                     equalTo( "500" ) );
+            assertThat(
+                    configurationAsMap.get( LdbcSnbBiWorkloadConfiguration.OPERATION_25_INTERLEAVE_KEY ),
+                    equalTo( "500" ) ); // TODO where does this number come from?
```

Comment:
TODO?

FYI, it's been a while but I think the factor 10x comes from: https://github.com/ldbc/ldbc_snb_driver/blob/e839ad08daafd298fabbbb533c9986a5396346cc/src/test/java/com/ldbc/driver/workloads/ldbc/snb/bi/BiWorkloadTest.java#L281